src/components/dashboard/FeedItems/ListEventItem.js

```diff
@@ -60,38 +55,63 @@ const ListEvent = ({ item: feed, theme, styles }: FeedEventProps) => {
 
 const getStylesFromProps = ({ theme }) => ({
   innerRow: {
-    padding: normalize(4),
+    alignItems: 'center',
     flexDirection: 'row',
     justifyContent: 'center',
-    alignItems: 'center',
+    padding: normalize(8),
     width: '100%',
   },
-  avatatBottom: {
+  avatarBottom: {
     alignSelf: 'flex-end',
   },
   mainSection: {
-    marginLeft: normalize(4),
+    marginLeft: normalize(8),
   },
   borderRow: {
+    alignItems: 'center',
     borderBottomStyle: 'solid',
     borderBottomWidth: normalize(2),
-    paddingBottom: normalize(4),
-    marginBottom: normalize(4),
+    display: 'flex',
+    justifyContent: 'space-between',
+    marginBottom: normalize(8),
+    paddingBottom: normalize(5),
```

Comment:
Same `theme.paddings` thing.

Particularly about `paddingBottom`, I'm not sure if it has to be `5` or a value multiple of `4`

Reply:
Changed `5` to `4`.